app/src/main/java/movierating/services/UserService.java

```diff
@@ -1,9 +1,10 @@
 package movierating.services;
 
+import movierating.exceptions.AddUserException;
 import movierating.models.User;
 import movierating.repositories.UserRepository;
 
 public interface UserService {
 
-    public void addUser(User user);
+    public void addUser(User user) throws AddUserException;
```

Comment:
I think you forgot to commit the file `AddUserException`.

Reply:
Have added the file.